### DISCUSSION TITLE
About ZS-TTS dataset selects

### DISCUSSION THREAD

**Discussion by Lizh-TW** [coqui-ai/TTS#1338] [General]
Hi According to my recent experiments because (figure 0), I have something experience about select the dataset on ZS-TTS.
  * figure 0 
![image](https://user-images.githubusercontent.com/58511222/156921176-17f3d631-5c2a-4ce2-8b0c-441148c3e996.png)

* First, I think the speaker have wav files count and the speaker's TTS similarity it not have a direct relationship(see figure 1)
  * figure 1 ![image](https://user-images.githubusercontent.com/58511222/156920005-1f3ee75a-3dd6-4612-9edc-c25a58c0cf48.png)
* Second, base the first, so I try to add Mozilla dataset in my training data, Then I got good result in unseendata. (See figure 2 data is my classmate. figure3 is the official YourTTS and use data is  VoxCeleb1)
  * figure 2 ![image](https://user-images.githubusercontent.com/58511222/156920334-3ea6f292-e1f5-4cee-8aec-b393f4fa22da.png)
  * figure 3 ![image](https://user-images.githubusercontent.com/58511222/156920361-c2a8e504-cf70-4c74-b99d-26f13ad2f0b2.png)
* Finally, I found that different accents need to be trained separately, I have tried training in Chinese (China and TW)

**Comment by erogol** [comment]
Thanks for sharing your results. Really interesting. 

One question. What did you use to measure speaker similarity?

**Reply by Lizh-TW** [reply]
@erogol Cosine similarity. 100 sentences of TTS, and 10 audio files are used count the center.

**Reply by erogol** [reply]
So you compare the speaker embeddings against the average speaker embedding?

**Reply by Lizh-TW** [reply]
@erogol Yes~

**Reply by 118dz** [reply]
有训练的数据和教程吗？我在这个卡住了不知道怎么弄，如何去构造相应的数据集

**Reply by Lizh-TW** [reply]
@118dz 甚麼意思呢?我想訓練的Dataset整理方式都差不多的吧?

**Reply by 118dz** [reply]
想学习一下，我自己下了一个中文的训练集，不知道咋训练，能教教我吗，不知道是环境不对还是怎么一回事

**Reply by Lizh-TW** [reply]
@118dz 你有嘗試過使用官方接學訓練一種單speaker模型嗎?先從這部分開始吧

**Reply by 118dz** [reply]
就是按照官方的那个教程下了训练集，但是吧感觉那个好像没跟新了，有些方法和变量它不适用了，然后就是一直报错

**Reply by Lizh-TW** [reply]
@118dz 抱歉那我可能無法幫助你，我已經近1年沒有碰此項目了，身邊也沒有足夠效能的電腦來測試。也許你可以把抱錯貼出來詢問作者

**Reply by 118dz** [reply]
那您当初训练出来的数据集还在吗？训练出来的数据集怎么加载到程序中，有没有当初的一些笔记啥的，真心想学习一下，

**Reply by 118dz** [reply]
哈哈，可算是把他那个demo跑起来了，环境有点问题高了，降低了版本就行了，参数给的不对也改了，但是还没开始训练中文，下载的训练集csv和demo中的csv感觉结构有点不一样，这个要怎么修改是该代码还是怎么怎么操作？想请教一下，还有就是训练结束后怎么使用该模型，谢谢大佬的回复
